openapi-v1.yaml

```diff
@@ -6158,3 +6298,140 @@ paths:
           description: error response
           schema:
             $ref: "#/definitions/Error"
+
+  /taskgraphs/{namespace}/log:
```

Comment:
I’m sorry I didn’t notice this before—I think it might be better to put `/taskgraphs/log/{namespace}` since the fundamental object type is a task graph log, which will be separate from a server-side task graph.

Reply:
Coming back to this I could actually see it either way—if you think /taskgraphs/whoever/logs/UUID is better than /taskgraphs/logs/whoever/UUID then that’s fine by me.

Reply:
Resolving. We decided that `{namespace}` should not be in this endpoint